webpack.common.js

```diff
@@ -65,11 +65,6 @@ const commonConfig = env => {
                     test: /\.(ttf|eot|svg|gif|png|jpg)(\?v=[0-9]\.[0-9]\.[0-9])?$/,
                     loader: 'file-loader?name=[path][name].[hash].[ext]'
                 },
-                {
-                    test: /\.json$/,
-                    loader: 'json-loader'
```

Comment:
`json-loader` package isn't installed so i don't think we need this, also there hasn't been a commit in this package since 2018